1.Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
import numpy as np
Z = 1.96
80 + 1.96 * 16/ np.sqrt(256)

81.96

In [2]:
80 - 1.96 * 16/ np.sqrt(256)

78.04

## [78.04; 81.96]

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,оценить истинное значение величины X при помощи доверительного интервала, покрывающего этозначение с доверительной вероятностью 0,95.

In [3]:
import numpy as np
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
M = a.sum() / a.size 
M

6.590000000000001

In [4]:
# найдем критерий Стьюдента при а/2 = 2,5% и степени свободы n-1 = 10-1
t = 2.262

In [5]:
# находим несмещенную дисперсию
var_ = ((a - M)**2).sum() / (a.size - 1)
var_

0.2032222222222223

In [10]:
M + t * np.sqrt(var_)/ np.sqrt(10)

6.912461744087575

In [11]:
M - t * np.sqrt(var_)/ np.sqrt(10)

6.267538255912426

##  [6.2675; 6.9125]

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметроказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [9]:
Z = 1.645
Zн = (17.5 - 17) / (2 / 10)
Zн

2.5

Верна альтернативная гипотеза H1 на уровне значимости 5%

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [13]:
M = 200
n = 10
a = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
M1 = a.sum() / a.size
M1

198.5

In [17]:
# найдем критерий Cтьюдента при a/2 = 0.5%, n-1 = 9
t = 3.25

In [15]:
# находим несмещенную дисперсию
var_ = ((a - M1)**2).sum() / (a.size - 1)
var_

19.833333333333332

In [16]:
tн = (198.5-200) / (np.sqrt(var_) / np.sqrt(n))
tн

-1.0651074037450896

Верна нулевая гипотеза H0 на уровне значимости 1%